docs/en/latest/how-to-build.md

```diff
@@ -117,6 +117,15 @@ Please refer to: [Installing Apache APISIX with Helm Chart](https://github.com/a
 
         Solution: set `LDAP_DIR` with `luarocks config` manually, for example `luarocks config variables.LDAP_DIR /usr/local/opt/openldap/`
 
+5. If you have no more need of the Apache APISIX runtime, uninstall it.
+
+```shell
+  # Uninstall apisix command
+  $ make uninstall
+```
+
+  Attention please, this operation will totally **remove** the related files.
```

Comment:
```suggestion
Attention please, this operation will totally **remove** the related files.
```

Reply:
line#127 aligns at line#97, so it starts with two space.